docs/guide/user-guide.md

```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL: 
+1. [DeepESDL JupyterLab](#deepesdl-jupyterlab)
+2. [DeepESDL xcube viewer](#deepesdl-xcube-viewer)
+3. [DeepESDL datacube resources](#deepesdl-datacube-resources)
+
+
+# DeepESDL JupyterLab
+
+## Basic usage
+
+This section provides a brief introduction for users to the basic features of
+the JupyterLab environment as deployed on the DeepESDL system. 
+For more in-depth documentation on the various components, see the links in the
+[‘Further information’ section](#further-information).
+
+### Logging in and starting the JupyterLab profile
+
+To use the DeepESDL JupyterLab environment, navigate to
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
+DeepESDL user, please use your GitHub account to log in. 
+If your Jupyter server is not already running, you may be presented
+with a menu of user JupyterLab profiles to use for your session; there might be 
+one or more JupyterLab profiles to choose from, depending on the computational 
+resources needs of your team. Please select an adequate JupyterLab profiles for 
+your current task; it might not always require the strongest computational 
+resources available.
+After choosing your environment, you will see a progress bar appear for a few 
+seconds while it is started for you. 
+The JupyterLab interface will then appear in your web browser, ready for
+use.
+
+
+### Choosing a JupyterLab profile
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profiles provides a particular combination of pre-installed 
+package versions and resource settings. 
+
+If you have already started your session and need to change JupyterLab profiles, 
+you can do this by selecting ‘Hub control panel’ from the ‘File’ menu within
```

Comment:
```suggestion
you can do this by selecting *Hub control panel* from the *File* menu within
```

To me, putting phrases in quotation marks is indicating either quotes, code string literals, verbatim speech, or ironic meaning. Therefore I prefer highlighting of UI elements via font style.